### PR TITLE
fix: Add descriptions to JSON schema

### PR DESCRIPTION
**Pull request by candiduslynx**
See https://github.com/invopop/jsonschema/pull/112 (https://github.com/cloudquery/jsonschema/pull/9)

Blocked by https://github.com/cloudquery/codegen/pull/60

### CODE REVIEW

File: .github/workflows/source_aws.yml [hunk 1]
```diff
@@ -50,6 +50,8 @@ jobs:
         uses: cloudquery/setup-cloudquery@v3
         with:
           version: v3.23.1
+      - name: vendor
```
Comment (erezrokah) [inline .github/workflows/source_aws.yml:53]: What's the reason we need this?

Comment (candiduslynx) [reply .github/workflows/source_aws.yml:53]: `gen` step (`make gen` -> `make gen-spec-schema`) requires code to be present to load comments from AWS SDK

Comment (candiduslynx) [reply .github/workflows/source_aws.yml:53]: I can move `go mod vendor` to `gen-spec-schema` step, it will have the same effect

**Review by erezrokah — APPROVED** [review]

File: .github/workflows/source_aws.yml (hunk 1 above)
Comment (candiduslynx) [reply .github/workflows/source_aws.yml]: Moved in 11e4350

**Pull request merged**